src/Components/App/App.js

```diff
@@ -82,33 +83,25 @@ const App = () => {
        console.log(error, "fetch")
       }
     }
-      
-
-      
-      useEffect(() => {
-        fetchData()
-  }
             
   useEffect(() => {
     fetchData()
-
     socket.on('receive_data', (data) => {
       setVendors([data.truck, ...data.updatedVendors])
     });
-
     return () => {
       socket.off('receive_data');
     };
   }, [])
 
   return (
-    <div>
-      <Header togView={toggleView} currentUser={currentUser}/>
```

Comment:
I noticed this passing of unnecessary data this morning, thanks for catching that.